diff -urN mc-4.8.8/misc/filehighlight.ini mc-4.8.8-patched/misc/filehighlight.ini
--- mc-4.8.8/misc/filehighlight.ini	2013-03-07 21:45:57.000000000 +1100
+++ mc-4.8.8-patched/misc/filehighlight.ini	2013-06-23 03:37:19.000000000 +1100
@@ -1,3 +1,7 @@
+[special]
+    type=SPECIAL
+    regexp=^\\.[^(\\.)]
+
 [executable]
     type=FILE_EXE
 
@@ -7,9 +11,6 @@
 [device]
     type=DEVICE
 
-[special]
-    type=SPECIAL
-
 [stalelink]
     type=STALE_LINK
 
diff -urN mc-4.8.8/misc/skins/default.ini mc-4.8.8-patched/misc/skins/default.ini
--- mc-4.8.8/misc/skins/default.ini	2013-03-07 21:45:57.000000000 +1100
+++ mc-4.8.8-patched/misc/skins/default.ini	2013-06-23 04:16:50.000000000 +1100
@@ -98,7 +98,7 @@
     _default_=lightgray;blue
     editbold=yellow;brightgreen
     editmarked=black;cyan
-    editwhitespace=brightblue;blue
+    editwhitespace=brightred;blue
     editlinestate=white;cyan
     bookmark=white;red
     bookmarkfound=black;green
